doc/changes/changes_0.6.14.md

```diff
@@ -0,0 +1,54 @@
+# Udf Debugging Java 0.6.14, released 2024-11-18
+
+Code name: Fix CVE-2024-47535: io.netty:netty-common:jar:4.1.108.Final:runtime
+
+## Summary
+
+This release fixes CVE-2024-47535 in `io.netty:netty-common:jar:4.1.108.Final:runtime`.
```

Comment:
Please note whether this is a test or production dependency.

Reply:
And maybe also if it's a direct or transitive dependency and if transitive then via which direct dependency.